modules/ROOT/pages/installation/linux/debian.adoc

```diff
@@ -282,7 +282,12 @@ For operating systems that are not using `systemd`, some package-specific option
 [[debian-service-start-automatically]]
 == Starting the service automatically on system start
 
-On Debian-based distributions, Neo4j is enabled to start automatically on system boot by default.
+On Debian-based distributions, users will need to run the following command to ensure that Neo4j starts automatically at boot time:
```

Comment:
We address the users as you and we try to avoid Future tense where possible.
My suggestion is:

On Debian-based distributions, run the following command to ensure that Neo4j starts automatically at boot time: